pubsub/utils.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pubsub
+
+// GetEntryIds provides list of entry Ids to be used for sending messages for bulk subscribe
+func GetEntryIds(n int) []int {
```

Comment:
`GetEntryIDs` to be consistent (surprised the linter didn't call this out)

Reply:
Changed entryID to not remain index based anymore. So, removed this part and corresponding heloer function in `pubsub`.